### ISSUE TITLE
App crashes at launch 

### ISSUE THREAD

**Issue by rickdps** [tito/kivy-gmaps#9]
There are no errors at all. It compiles, installs, but doesn't launch. Any thoughts ?

**Comment by moonpyx** [comment]
Hello, look, I had that problem and I solved it by analyzing the log from adb, which I found esque the create_polyline function of the gmaps __init__ file causes the crash, which did not implement it and the start app, however the map came out black and I do not know if it was for the apikey :(.

**Comment by rickdps** [comment]
Hey, thanks for responding. Could you please tell me how to get to this log you mentioned.  I use PyCharm on Ubuntu.

**Comment by moonpyx** [comment]
Of course, in the terminal with the command adb logcat -s (your app name): D python: D and your device connected in debug mode start your app (gmaps) so you can see the ones from java to gmaps where you will find the reason of kill activity for the reason given above.

**Comment by rickdps** [comment]
I really appreciate your help. Yeah I'm gonna check that out later and hopefully get this thing off the ground successfully. God bless!
